coconut/configuration/configuration.go

```diff
@@ -215,10 +277,65 @@ func formatListOutput( cmd *cobra.Command, output []string)(parsedOutput []byte,
 	return parsedOutput, nil
 }
 
-func isInputNameValid(input string)(valid bool) {
-	if strings.Contains(input, "/") || strings.Contains(input, "@") {
-		return false
-	} else {
-		return true
+func IsInputNameValid(input string) bool {
+	return InputRegex.MatchString(input)
+}
+
+func IsInputSingleValidWord(input string) bool {
+	return !strings.Contains(input, "/") && !strings.Contains(input, "@")
+}
+
+// Method to return the latest timestamp for a specified component & entry
+// If no keys were passed an error and code exit 3 will be returned
+func GetLatestTimestamp(keys []string, component string, entry string)(timestamp string, err error, code int) {
+	keyPrefix := componentsPath + component + "/" + entry
+	if len(keys) == 0 {
+		err = errors.New(fmt.Sprintf("No keys found"))
+		return "", err, 3
+	}
+
+	var maxTimeStamp string
+	for _, key := range keys {
+		componentTimestamp := strings.TrimPrefix(key, keyPrefix + "/")
+		fmt.Println(componentTimestamp)
+		if strings.Compare(componentTimestamp, maxTimeStamp) > 0 {
```

Comment:
This (and similar situations, if any) should be an integer comparison, not a string comparison.

Reply:
👍